website/docs/reference/datasources/hubspot/README.md

```diff
@@ -0,0 +1,152 @@
+
+# HubSpot
+
+
+HubSpot is a **cloud-based CRM** designed to help align sales and marketing teams, promote sales enablement, increase ROI, and improve your inbound marketing strategy to produce more qualified leads. In simple words, it is a software platform created to assist your business with marketing and sales.
+
+:::info
+To integrate Hubspot with Appsmith, you should understand the [basics of connecting to databases](core-concepts/connecting-to-data-sources/) on Appsmith.
+:::
+
+Appsmith provides an easy way to integrate with HubSpot, and enables you to perform different operations using a custom UI built with minimal configurations.
+
+
+# Create HubSpot Datasource
+
+To add a HubSpot API, navigate to **Explorer** >> Click on the **+** icon (next to Datasources) >> Select **HubSpot API** under APIs.
+
+<figure>
+ <object data="https://www.youtube.com/embed/h2Jp8PenbTM" width='750px' height='400px'></object>
+ <figcaption align="center"><i>Create Hubspot Datasource</i></figcaption>
+</figure>
+
+
+
+## Connection Settings
+
+Configure the Hubspot API as illustrated below:
+
+![](/img/hubspot-connection.png)
+
+
+
+### Authentication Type
+Appsmith allows you to select from the available authentication types to integrate with an Hubspot database:
+
+#### **Bearer Token** 
+
+Bearer Tokens are the predominant access token used with OAuth 2.0.
+This is a single string that is the authentication of the API request, sent in an HTTP “Authorization” header. You can authenticate to the API by providing your API key in the HTTP authorization bearer token header. 
```

Comment:
Do you mean? (You could also authenticate the API by providing the API Key in the HTTP authorization bearer token header.)